EMS/src/GUI/UserDashboard.java

```diff
@@ -48,7 +48,7 @@ public UserDashboard(User.UserType userType, String username) {
     private void greetUser() {
         // Capitalizing first letter of userName
         String userName = user.getFirstName();
-        userName = StringUtils.capitalize(userName);
+        //userName = StringUtils.capitalize(userName);
```

Comment:
This line is used to capitalize the first letter of the username to great the user properly, Why is it commented?

Reply:
It was for testing purpose.